megamek/src/megamek/common/TechConstants.java

```diff
@@ -760,6 +760,23 @@ public static int getCockpitTechLevel(int cockpitType, long entityType,
                     } else {
                         return T_IS_TW_NON_BOX;
                     }
+                //Same as Torso-Mounted
+                case Mech.COCKPIT_TORSO_MOUNTED_INDUSTRIAL:
```

Comment:
If this is truly the same as Torso-mounted, you should instead place just the case statement with the Torso Mounted (non-Industrial) code to avoid duplication.

Reply:
Got it. I was did that just because it will follow the order, but it seems you are right.